packages/test-utils/src/scripts/check-balances.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * This is a stand-alone script that looks an address' balances
+ */
+
+import { password } from '@inquirer/prompts';
+import { Provider, WalletUnlocked } from 'fuels';
+
+let { L2_ADDRESS, L2_RPC } = process.env;
+
+const main = async () => {
+  const provider = await Provider.create(L2_RPC, { resourceCacheTTL: -1 });
+
+  if (!L2_ADDRESS) {
+    const privKey = await password({ message: 'Enter private key' });
```

Comment:
nit: why get a privatekey instead of address?

Reply:
So that you can get information about the loaded address from a certain private key